en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -6,190 +6,154 @@ iconBrand: php
-As opposed to other variables, these should be declared as lower case with underscores.
-
-
-```php
-class MyClass
-{
-    private static $my_config_variable = 'foo';
-}
-```
-
-## Prefer identical (===) comparisons over equality (==)
-
-Where possible, use type-strict identical comparisons instead of loosely typed equality comparisons.
-Read more in the PHP documentation for [comparison operators](https://php.net/manual/en/language.operators.comparison.php) and [object comparison](https://php.net/manual/en/language.oop5.object-comparison.php).
-
-
-```php
-// good - only need to cast to (int) if $a might not already be an int
-if ((int)$a === 100) {
-    doThis();
-}
-
-// bad
-if ($a == 100) {
-    doThis();
-}
-```
-
-## Separation of Logic and Presentation
-
-Try to avoid using PHP's ability to mix HTML into the code.
-
-
-```php
-// PHP code
-public function getTitle() 
-{
-    return "<h2>Bad Example</h2>";
-}
-```
-
-```ss
-// Template code
-$Title
-```
-
-Better: Keep HTML in template files:
-
-
-```php
-// PHP code
-public function getTitle() 
-{
-    return "Better Example";
-}
-```
-
-```ss
-// Template code
-<h2>$Title</h2>
-```
-
-## Comments
-
-Use [phpdoc](https://phpdoc.org/) syntax before each definition (see [tutorial](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_phpDocumentor.quickstart.pkg.html)
-and [tag overview](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.pkg.html)).
-
- * Methods should include at least `@param` and `@return`.
- * Include a blank line after the description.
- * Use `{@link MyOtherClass}` and `{@link MyOtherClass->otherMethod}` for inline references.
- * Denote preformatted code examples in `<code></code>` blocks.
- * Always start block-level comments containing phpdoc with two asterisks (`/** ... */`).
-
-Example:
-
-
-```php
-/**
- * My short description for this class.
- * My longer description with
- * multiple lines and richer formatting.
- *
- * Usage:
- * <code>
- * $c = new MyClass();
- * $c->myMethod();
- * </code>
- *
- * @package custom
- */
-class MyClass extends Class
-{
-    /**
-     * My Method.
-     * This method returns something cool. {@link MyParentMethod} has other cool stuff in it.
-     *
-     * @param string $colour The colour of cool things that you want
-     * @return DataList A list of everything cool
-     */
-    public function myMethod($colour)
-    {
-        // ...
-    }
-
-}
-```
+- Aim for [PSR-12 Coding Standards](https://www.php-fig.org/psr/psr-12/) compliance.
+- Prioritise the human readability of code over "clever" code that reduces the number of lines at the expense of readability.
+- All symbols and documentation should use UK-English spelling (e.g. "behaviour" instead of "behavior"), except when necessitated by third party conventions (e.g using PHP's `Serializable` interface).
+- Avoid writing HTML in PHP code. Instead, put HTML is template files.
```

Comment:
It's kind of wrong in this context since that has more to do with the branch that a pull-request is targeting, rather than things like method naming - for instance it's not something that a linter could pickup if we just ran the linter on some files on our laptop